### DISCUSSION TITLE
I absolutely hate the dashboard and I consider it one of the most useless out of all online project management tools I use.

### DISCUSSION THREAD

**Discussion by W-Lawless** [community/community#17120] [Other Feature Feedback, Questions, & Ideas]
There is absolutely nothing on it that is relevant or helpful to my daily workflow. I see commits to repos I am _following_ in a feed. Why would I care about this? It's geared towards use of Github as a social media, exploratory tool, I get it. It is supposed to encourage me to go out and explore repos, contribute open source, keep up with other devs in repos , etc

But that is entirely antithetical to what I actually care about. I dont want to see a feed of COMMITS - even if I was interested in repos im *following* (vs the ones ive authored or my organization has authored) 

I am using git to manage source control in a professional organization with several other organization's workspaces. I WANT TO SEE MY WORKSPACES AND REPOSITORIES. I DONT WANT TO CLICK A LITTLE, ITTY BITTY DROP DOWN TO SEE THIS. The list of repos underneath here with search bar, usually shows MY repos and not my organizations repos. Again, this may be here, but it is NOT VISUAL. It is TEXT. A TEXT input and a TEXT list. The oppurtunity for VISUAL COMMUNICATION - 90% of my screen real estate - is  again devoted to a COMMIT FEED. Guys. a COMMIT FEED IS A LIST OF UGLY, CRYPTIC IDS. commit 28asdan commit 389a commit 909asd commit 198ajds -- WHY ON EARTH IS SOMETHING SO HARD TO GET A SENSE OF VISUALLY 90% OF MY SCREEN. 

This UI (and UX) is seriously awful, to the point I'm writing this polemic about it.

A commit feed is a cryptic & high cognitive overhead component to grok. I don't want to see it UNLESS I WANT TO SEE IT. It reads visually as huge blocks of text where one dev commits 5-7 items that are all labeled obscurely and it is impossible to get an 'at a glance' idea of what is going on from a verbose commit feed. IF YOU GUYS WANT TO INCLUDE THIS ON THE DASHBOARD, INCLUDE THE HEARTBEAT SENSORS FOR THE REPOS COMMIT HISTORY WITH MAYBE A BIT MORE UI QUES WORKED IN , OR USE THE CHECKERBOARD COMPONENT UNDERNEATH THE REPO 

My god, seriously guys this dashboard makes me so mad, it is so entirely useless and as one of the leading professional development platforms in the industry I am at a loss about how exactly anyone approved a dashboard that is 95% useless screen real estate.

Show my me organizations, other organizations involved with mine, our repos, the most recently updated ones first, with a quick checkerboard history or heartbeat history of the commit feed. Make it EASY and QUICK for me to get to the repo I want to see without me digging through 3-4-5 interactions with stupid little, sequestered and teeny tiny drop downs. I DONT WANT TO CLICK THE DROP DOWN. I DONT WANT TO USE THE SEARCH FILTER.  I DONT WANT TO CLICK THROUGH TO MY ORGANIZATION 'HOME' PANEL AND THEN TO THE REPO AND THEN TO THIS. Thats 3-4-5 interactions to get to WHAT SHOULD BE FRONT AND CENTER ON MY DASHBOARD

PLEASE PLEASE FIX THIS OR IM PUSHING OUR ORGANIZATION TO GO TO BITBUCKET 100000%

**Comment by mastercoms** [comment]
I personally like the dashboard, it helps me find interesting new things that help me in development.

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Reply by petedegraw** [reply]
That's great until they change it and you have more stuff showing in the feed.

**Reply by FrostyX** [reply]
Which still sucks because the only thing it adds is filtering for a feed that OP doesn't care about in the first place. Disabling the feed doesn't help absolutely anything because now the 90% of screen estate is simply used for **nothing**.

Also, the only thing I remotely care about in the entire feed, since it is forcefully shoved to our faces, is the slight dopamine hit from seeing that somebody starred my repo. But I see them only in the old feed. The new feed implementation doesn't show this even though I don't filter anything out.

**Reply by W-Lawless** [reply]
> there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters
> 
> ![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

This is definitely a step in the right direction, and I've noticed that the left-hand sidebar now shows me pretty much all the repos I'm associated with instead of having them hidden in dropdown menus as it was last year when I opened this issue, which is way, way better. The old component that would only show me *my personal* repos and then hide everything else in click 1 - 2 - 3 - 4 etc was seriously so aggravating. 

That being said, my 'ideal' "feed" isn't so much a feed of realtime updates, so much as it is just a list.

IE: Take the sidebar on the left, make it front and center, expand each item on there to be a full card, include the checkerboard commit history to visually indicate chronological activity/interest, and maybe some other metrics idk - maybe some quick links along the bottom of the card to get to issues / commits / prs / whatever 

There's no need to reinvent the wheel here. Just come up with some components that *reduce the navigation overhead* 

I understand the focus of github as being a site to foster social, open source development. You needn't scrap the whole dashboard. Just.. put another tab alongside 'For You' - call it....  'Repositories' ? 

I know this isn't my personal suggestion box but seriously... embed some analytics triggers in a new component like this, role it out on some randomized A/B testing, make sure you properly UI cue people to notice the new feature... and measure how much engagement it gets.

I'd think, that it would be widely used and much appreciated by a large portion of users

**Comment by elias-io** [comment]
Its 2024 and the dashboard makes me so frustrated everytime I open Github. 
Why do I open Github? 
1. To quickly go to my repositories.
2. To check my last Github actions.
3. To quickly check the latest pull-requests. 
There needs to be a chrome extension which modifies the Github Dashboard for you and makes it not TOTAL SHIT!!

**Reply by FrostyX** [reply]
I don't access github.com anymore, I made a bookmark for my GitHub profile and put all relevant links there. The dashboard is just useless.

**Comment by michaeloa** [comment]
Two years later, and the dashboard remains useless. Everyone in my organization hates it, and actively avoids using it.

As elias notes, the key actions are quite simple - and the dashboard helps with none of those.

**Comment by wrigleyster** [comment]
There is nothing useful on the Dashboard. It is so bad. Fine, you want the default experience to be one that helps people find repos. But why in all these years the Dashboard wasn't made configurable, allowing people to add repo list, pull-requests, projects, etc. and remove unnecessary widgets, is beyond me. Even just an option to redirect to the profile page upon login would be helpful.

**Comment by leo-cheron** [comment]
Please Github, do something about it. This dashboard is wasting time of millions of people every day.
